pkg/skaffold/build/util.go

```diff
@@ -45,3 +51,16 @@ func MergeWithPreviousBuilds(builds, previous []Artifact) []Artifact {
 
 	return merged
 }
+
+func TagWithDigest(tag, digest string) string {
+	return tag + "@" + digest
```

Comment:
to know if "tag@digest" or "@digest" will be the way we would need to pass in context here would we not ? then have a way to auto-guess wether the tagwithdigest strategy should be with or without the tag.

Reply:
not quite sure I understand what you mean here? we shouldn't need to care whether or not a tag is provided here. this PR isn't actually changing anything about the tagging logic in skaffold

Reply:
@maxandersen Another way to do it would be to make `TagWithDigest` a variable and to change its implementation early in the code path, when we try to detect the type on tube context we are connected to.

Reply:
okey, will see if i can find some time over the break to give it a go and see if things gets solved this way.